enacts/crop_suitability/maproom_crop_suit.py

```diff
@@ -0,0 +1,545 @@
+    tmin_mrg_season = tmin_mrg_year.sel(T=tmin_mrg_year["T.season"] == target_season)
+    tmax_mrg_year = tmax_mrg.sel(T=tmax_mrg['T.year']==target_year)
+    tmax_mrg_season = tmax_mrg_year.sel(T=tmax_mrg_year["T.season"] == target_season)
+
+    if data_choice == "suitability_map":
+        crop_suit_vals = crop_suitability(
+            rr_mrg_season, min_wet_days, wet_day_def, tmax_mrg_season, tmin_mrg_season, 
+            lower_wet_threshold, upper_wet_threshold, maximum_temp,
+            minimum_temp, temp_range, target_season 
+            ) 
+        data_tile = crop_suit_vals["crop_suit"]
+    else:
+        data_var = CONFIG["map_text"][data_choice]["data_var"]
+        if data_choice == "precip_map":
+            data_tile = rr_mrg_season[data_var]
+        if data_choice == "tmin_map":
+            data_tile = tmin_mrg_season[data_var]
+        if data_choice == "tmax_map":
+            data_tile = tmax_mrg_season[data_var]
+    if (
+            # When we generalize this to other datasets, remember to
+            # account for the possibility that longitudes wrap around,
+            # so a < b doesn't always mean that a is west of b.
+            x_min > data_tile['X'].max() or
+            x_max < data_tile['X'].min() or
+            y_min > data_tile['Y'].max() or
+            y_max < data_tile['Y'].min()
+    ):
+        return pingrid.image_resp(pingrid.empty_tile())
+
+    data_tile = data_tile.sel(
+        X=slice(x_min - x_min % RESOLUTION, x_max + RESOLUTION - x_max % RESOLUTION),
+        Y=slice(y_min - y_min % RESOLUTION, y_max + RESOLUTION - y_max % RESOLUTION),
+    ).compute()
+
+    mycolormap = CMAPS["rainbow"]
+
+    if data_choice == "suitability_map":
+        mymap = data_tile
+    else:
+        mymap = data_tile.mean("T")
+
+    mymap = np.squeeze(mymap)
+    mymap.attrs["colormap"] = mycolormap
+    mymap = mymap.rename(X="lon", Y="lat")
+    mymap.attrs["scale_min"] = mymap_min
+    mymap.attrs["scale_max"] = mymap_max
+    print(mymap)
+    result = pingrid.tile(mymap, tx, ty, tz, clip_shape)
+
+    return result
+
+
+@APP.callback(
+    Output("colorbar", "children"),
+    Output("colorbar", "colorscale"),
+    Output("colorbar", "max"),
+    Output("colorbar", "tickValues"),
+    Input("data_choice", "value"),
+)
+def set_colorbar(
+    data_choice,
+):
+    mymap_max = CONFIG["map_text"][data_choice]["map_max"]
+    if data_choice == "suitability_map":
+        tick_freq = 1
+    else:
+        tick_freq = 5
+    return (
+        f"{CONFIG['map_text'][data_choice]['menu_label']} [{CONFIG['map_text'][data_choice]['units']}]",
+        CMAPS["rainbow"].to_dash_leaflet(),
+        mymap_max,
+        [i for i in range(0, mymap_max + 1) if i % tick_freq == 0],
+        #[0,1,2,3,4,5],
+    )
+
+if __name__ == "__main__":
+    APP.run_server(
+        GLOBAL_CONFIG["dev_server_interface"],
+        GLOBAL_CONFIG["dev_server_port"],
+        debug=GLOBAL_CONFIG["mode"],
```

Comment:
Should be
```
debug = GLOBAL_CONFIG["mode"] != "prod"
```
or copy the more complex logic from app.py, which also turns warnings into errors when in debug mode so we don't miss them.

Reply:
Is it preferred that I use the more complex logic? I did not add it in when I just pushed the correction but I can add it if it is preferred.